recipes/net-snmp/net-snmp.inc

```diff
@@ -62,4 +73,5 @@ RDEPENDS_${PN}-client += "${AUTO_PACKAGE_UTILS_PROVIDES}"
 
 FILES_${PN}-mibs = "${datadir}/snmp/mibs"
 FILES_${PN}-client += "${datadir}/snmp"
-FILES_${PN}-server += "${sysconfdir} ${sbindir}/*"
+FILES_${PN}-server += "${sysconfdir}/init.d ${sbindir}/*"
+FILES_${PN}-conf += "${sysconfdir}/snmp"
```

Comment:
you should probably set up RDEPENDS to pull this in when someone just pulls in server or client